integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -726,21 +729,34 @@ void testUpdateImageName() {
     String imageUpdate = KIND_REPO != null ? KIND_REPO
         + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
         : TEST_IMAGES_REPO + "/" + WEBLOGIC_IMAGE_NAME_DEFAULT + ":" + imageTag;
-    getLogger().info(" The image name used for update is: {0}", imageUpdate);
+    getLogger().info(" The image name used for the 1st update is: {0}", imageUpdate);
+
+    //1st time update image and restartVersion without image tagging/pushing
+    patchDomainWithNewImage(imageUpdate);
+    String newRestartVersion = patchDomainResourceWithNewRestartVersion(domainUid, introDomainNamespace);
+    logger.log(Level.INFO, "New restart version is {0}", newRestartVersion);
+    // verify the server pods are not restarted because newImage is not tagged, not pushed to ocir
+    //admin server pod will be in pending state(ImagePullBackOff)
+    logger.info("Verifying restart did NOT occur for domain {0} in namespace {1}",
+        domainUid, introDomainNamespace);
+    assertThrows(ConditionTimeoutException.class, () -> {
+      verifyRollingRestartOccurred(cl1podsWithTimeStamps, 1, introDomainNamespace);
+    });
+    if (cluster2Created) {
+      assertThrows(ConditionTimeoutException.class, () -> {
+        verifyRollingRestartOccurred(cl2podsWithTimeStamps, 1, introDomainNamespace);
+      });
+    }
+
+    //2nd time image update
+    imageTag = CommonTestUtils.getDateAndTimeStamp();
+    imageUpdate = KIND_REPO != null ? KIND_REPO
+        + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
+        : TEST_IMAGES_REPO + "/" + WEBLOGIC_IMAGE_NAME_DEFAULT + ":" + imageTag;
+    getLogger().info(" The image name used for the 2nd update is: {0}", imageUpdate);
     dockerTag(imageName, imageUpdate);
     dockerLoginAndPushImageToRegistry(imageUpdate);
```

Comment:
Make the image available by pushing into OCIR with proper tagging

Reply:
Done